sonar-db/src/test/java/org/sonar/db/component/ComponentDaoTest.java

```diff
@@ -51,16 +51,17 @@
 
   @Rule
   public DbTester db = DbTester.create(System2.INSTANCE);
+  ComponentDbTester componentDb = new ComponentDbTester(db);
 
-  DbSession dbSession = db.getSession();
+  DbSession session = db.getSession();
```

Comment:
Personally I prefer using the term `dbSession` as it removes ambiguity with `UserSession`